app/models/setting.rb

```diff
@@ -83,4 +88,32 @@ def self.invoice_reminder_in_days
   def self.wishlist_size
     Setting.find_by(code: :wishlist_size).value.to_i
   end
+
+  def self.violations_count_regulations_link
+    hash = Setting.find_by(code: :violations_count_regulations_link)&.value
+    hash.present? ? JSON.parse(hash).with_indifferent_access[I18n.locale] : nil
+  end
+
+  private
+
+  def validate_value
+    methods = FORMAT_VALIDATIONS[code]
+    return unless methods
+
+    methods.each do |method|
+      send(method)
```

Comment:
Why `send` is needed here? This looks magical and doesn't communicate what this code does and why at a high level.

Reply:
Renamed variables here to make more sense.